### PR TITLE
PrinterPage: port to Granite.SimpleSettingsPage

### PR DESCRIPTION
**Pull request by danrabbit**
Automatically handles some things for us :)

### CODE REVIEW

File: src/PrinterPage.vala [hunk 1]
```diff
@@ -38,40 +44,22 @@ public class Printers.PrinterPage : Gtk.Grid {
...
-        attach (stack_switcher, 0, 1, 4, 1);
-        attach (stack, 0, 2, 4, 1);
-        attach (action_area, 0, 3, 4, 1);
-        show_all ();
+        printer.bind_property ("info", this, "title");
+        printer.bind_property ("location", this, "description");
 
-        printer.bind_property ("info", title, "label");
+        status_switch.active = printer.state != "5" && printer.is_accepting_jobs;
+        status_switch.bind_property ("active", printer, "enabled", GLib.BindingFlags.BIDIRECTIONAL);
```
Comment (jeremypw) [inline src/PrinterPage.vala]: If you add the flag GLib.BindingFlags.SYNC_CREATE then the previous line should not be necessary.

Comment (danrabbit) [reply src/PrinterPage.vala]: Yeah probably better to be consistent here and make sure that the switch state always means the same thing :+1:

File: src/PrinterPage.vala [hunk 2]
```diff
@@ -56,8 +56,7 @@ public class Printers.PrinterPage : Granite.SimpleSettingsPage {
         printer.bind_property ("info", this, "title");
         printer.bind_property ("location", this, "description");
 
-        status_switch.active = printer.state != "5" && printer.is_accepting_jobs;
-        status_switch.bind_property ("active", printer, "enabled", GLib.BindingFlags.BIDIRECTIONAL);
+        status_switch.bind_property ("active", printer, "enabled", GLib.BindingFlags.SYNC_CREATE);
```
Comment (jeremypw) [inline src/PrinterPage.vala]: You will want to keep it BIDIRECTIONAL? So the flag parameter is
` BindingFlags.BIDIRECTIONAL | BindingFlags.SYNC_CREATE`

Comment (danrabbit) [reply src/PrinterPage.vala]: Oops, you're right. Fixed!

**Pull request merged**